flink-python/pyflink/table/utils.py

```diff
@@ -67,3 +67,7 @@ def tz_convert_to_internal(s, t: DataType, local_tz):
         elif is_datetime64tz_dtype(s.dtype):
             return s.dt.tz_convert(local_tz).dt.tz_localize(None)
     return s
+
+
+def exec_insert_table(table, table_path) -> JobExecutionResult:
```

Comment:
What about moving to `test_case_utils`?

Reply:
@dianfu To regard `exec_insert_table ` as util method.

Reply:
This method is only used in test cases and so I prefer to move it to test_case_utils. I see it's also used in word_count.py, however, I suggest not use it in word_count.py. This method isn't a public interface and we'd better not exposing it to users.